.github/actions/webapp/action.yml

```diff
@@ -0,0 +1,116 @@
+name: 'Deploy Azure Functions'
+description: 'This custom GitHub Actions module deploys from defined source to defined Azure Functions microservice'
```

Comment:
This file is called webapp/action.yml but looks like it's duplicating the azure functions deployment